tst/EnergyPlus/unit/SingleDuct.unit.cc

```diff
@@ -2546,3 +2546,89 @@ TEST_F( EnergyPlusFixture, TerminalUnitMixerInitTest ) {
 	DataHeatBalance::ZoneIntGain.deallocate( );
 
 }
+
+TEST_F( EnergyPlusFixture, VAVReheatTerminal_SizeMinFrac ) {
+	std::string const idf_objects = delimited_string( {
+		"Version,8.4;",
```

Comment:
Version 8.4?  :smiley:

Reply:
What? It works.  Will change if I change the other line.